src/actions/javascript/rekeyAction.ts

```diff
@@ -0,0 +1,9 @@
+import { KmsClient } from "cloudproof_js";
+import { BACKEND_URL } from "./backendConfig";
+//
+// Perform a rekey operation
+//
+export const rekeyAccessPolicy = async (kmsToken: string, masterSecretKeyUId: string): Promise<string[]> => {
+  const client = new KmsClient(BACKEND_URL, kmsToken);
+  return await client.rekeyCoverCryptAccessPolicy(masterSecretKeyUId, "country::Germany");
```

Comment:
same as before. It induces errors on the KMS with a capital letter ( or maybe I messed up with my tests maybe try it so that we make sure for good ? )